.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: test
+on: push
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    container: golang:1.15
```

Comment:
Does the alpine container work here as well?  Maybe not?

Reply:
The `alpine` image is missing a few packages like `gcc` and `libc`...I used the standard golang image so I didn't have to add that installation to the workflow